user_manual/changes/SB06-030.yaml

```diff
@@ -0,0 +1,11 @@
+type: new-feature
+title: Add a command line app framework for Ada API
+
+description: |
+    Add a new package `Libadalang.Helpers`, with a generic package `App`, that
```

Comment:
This field contains ReST-formatted text, so the backquotes should be doubled.

Reply:
Thanks